environment.yml

```diff
@@ -19,6 +20,7 @@ dependencies:
   - pandas=1.3.4
   - panel=0.12.4
   - param=1.11.1
+  - pint-xarray
```

Comment:
Is pint-xarray a direct dependency? I'm not seeing it imported anywhere. If it is, then the same comment applies about version pinning, otherwise it should be removed.

Reply:
I will double check though

Reply:
You're right, I can delete it! Thanks for the sanity check.